draft-ietf-tls-dtls-connection-id.md

```diff
@@ -171,7 +184,7 @@ connections which have negotiated this extension use a distinguished
 record type tls12_cid(25). Use of this content type has the following
 two implications:
 
-- The CID field is present
+- The CID field is present in the record
 - The true content type is inside the encryption envelope, as described
   below.
```

Comment:
+ - The MAC calculation follows the process described in {{}}.

Reply:
updated text